pac/Jenkins-marketplace-build.groovy

```diff
@@ -0,0 +1,142 @@
+                    script {    
+                        def cmd = "docker ps --format '{{.Image}}'"
+                        def cmd2 = "docker ps --format '{{.Names}}'"
+                        def image = sh(returnStdout: true, script: cmd).trim()
+                        def name  = sh(returnStdout: true, script: cmd2).trim()
+                        sh '''$(aws ecr get-login --no-include-email)'''
+                        [image.tokenize(), name.tokenize()].transpose().each { x ->
+                            sh """ docker tag ${x[0]} ${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_DEFAULT_REGION}.amazonaws.com/evolved5g:${x[1]}-${VERSION}"""
+                            sh """ docker tag ${x[0]} ${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_DEFAULT_REGION}.amazonaws.com/evolved5g:${x[1]}-latest"""
+                            sh """ docker image push ${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_DEFAULT_REGION}.amazonaws.com/evolved5g:${x[1]}-${VERSION}"""
+                            sh """ docker image push ${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_DEFAULT_REGION}.amazonaws.com/evolved5g:${x[1]}-latest"""
+                        }
+                    }
+                }
+            }
+        }
+
+        stage('Modify container name to upload Docker-compose to Artifactory') { 
+            steps {
+                withCredentials([usernamePassword(credentialsId: 'docker_pull_cred', usernameVariable: 'ARTIFACTORY_USER', passwordVariable: 'ARTIFACTORY_CREDENTIALS')]) {
+                    script {   
+                        sh ''' docker login --username ${ARTIFACTORY_USER} --password "${ARTIFACTORY_CREDENTIALS}" dockerhub.hi.inet '''
+                        def cmd = "docker ps --format '{{.Image}}'"
+                        def cmd2 = "docker ps --format '{{.Names}}'"
+                        def image = sh(returnStdout: true, script: cmd).trim()
+                        def name  = sh(returnStdout: true, script: cmd2).trim()
+                        sh '''$(aws ecr get-login --no-include-email)'''
+                        [image.tokenize(), name.tokenize()].transpose().each { x ->
+                            sh """ docker tag "${x[0]}" dockerhub.hi.inet/evolved-5g/marketplace/"${x[1]}":${VERSION}"""
+                            sh """ docker tag "${x[0]}" dockerhub.hi.inet/evolved-5g/marketplace/"${x[1]}":latest"""
+                            sh """ docker image push --all-tags dockerhub.hi.inet/evolved-5g/marketplace/"${x[1]}" """
+                        }
+                    }
+                }               
+            }
+        }   
+        stage('Cleaning docker images and containers') {
+            steps {
+                catchError(buildResult: 'SUCCESS', stageResult: 'UNSTABLE') {
+                    sh '''
+                    docker stop $(docker ps -q)
+                    docker system prune -a -f --volumes
+                    sudo rm -rf $WORKSPACE/
+                    '''
+                }
+            }
+        }
+    }
+// post {
```

Comment:
are you sure this post stage should be commented? we need to always clean workspace, because next build in most of cases fails if it's not clean